src/main/java/org/edumips64/core/is/FPArithmeticInstructions.java

```diff
@@ -24,10 +24,19 @@
 
 package org.edumips64.core.is;
 
-import org.edumips64.core.*;
-import org.edumips64.core.fpu.*;
-//per diagnostica
+import org.edumips64.core.Converter;
+import org.edumips64.core.FCSRRegister;
+import org.edumips64.core.IrregularStringOfBitsException;
+import org.edumips64.core.IrregularWriteOperationException;
+import org.edumips64.core.MemoryElementNotFoundException;
+import org.edumips64.core.fpu.FPDivideByZeroException;
+import org.edumips64.core.fpu.FPInstructionUtils;
+import org.edumips64.core.fpu.FPInvalidOperationException;
+import org.edumips64.core.fpu.FPOverflowException;
+import org.edumips64.core.fpu.FPUnderflowException;
+import org.edumips64.core.fpu.RegisterFP;
 
+//per diagnostica
```

Comment:
You can remove this comment, it looks obsolete.

Reply:
Sure! No problem, shall I remove all occurrences of that comment across all classes?